chain/jsonrpc-adversarial-primitives/Cargo.toml

```diff
@@ -21,3 +21,4 @@ ser_de = [
   "near-jsonrpc-primitives",
   "near-network/test_features"
 ]
+deepsize_feature = ["deepsize", "near-primitives/deepsize_feature"]
```

Comment:
Sadly, I don't know a way to make `deepsize` name here "just work".

Reply:
@matklad Yes, that's unfortunate.